lib/ansible/module_utils/facts/sysctl.py

```diff
@@ -31,7 +31,7 @@ def get_sysctl(module, prefixes):
 
         try:
             rc, out, err = module.run_command(cmd)
-        except (IOError, OSError) as e:
+        except OSError as e:
```

Comment:
This and the following ones inconsistently don't turn e into ex. So maybe either don't do renaming at all or add some consistency. I'm still a believer that these contextual references should be named with words instead of 1–2 keystrokes.

Reply:
Yes, I am still working through the remaining parts of the PR. Re: consistency, this PR doesn't touch all exception handling in the codebase so it won't be consistent anyway. I am renaming (to be consistent with the new post DT code) just those that this PR is already changing.